PKHeX.WinForms/Program.cs

```diff
@@ -93,6 +94,12 @@ private static void CurrentDomain_UnhandledException(object sender, UnhandledExc
                 // Todo: make this translatable
                 ErrorWindow.ShowErrorDialog("An unhandled exception has occurred.\nPKHeX must now close.", ex, false);
             }
+            catch (MissingMethodException)
+            {
+                // Todo: make this translatable
+                MessageBox.Show(".NET Framework 4.6 needs to be installed for this version of PKHeX to run.", "PKHeX Error", MessageBoxButtons.OK, MessageBoxIcon.Stop);
+                Process.Start(@"https://www.microsoft.com/en-us/download/details.aspx?id=48130");
```

Comment:
We should keep it on 4.6, since it comes with some new features. Windows XP support is no longer a priority, making it a bit easier.

The error for 3.5 is descriptive enough that we shouldn't have to worry about it.  The fact that the window shows in 4.5 means it's possible to display a nicer error, probably the same one from this PR, maybe even a varient of my ErrorWindow to allow copy/pasting the URL into the browser.  The real question is about 4.0.  Given that it normally throws a MissingMethodException, I'm guessing that it fails right on Process.Start?  If it shows the MessageBox, then that's probably plenty.  If not, we'll have to get creative, or make it painfully obvious in the forums that the .Net Framework 4.6 is needed.